### PR TITLE
Code grooming

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: Symfony/CS/Fixer/PSR2/VisibilityFixer.php [hunk 1]
```diff
@@ -123,7 +123,7 @@ private function grabAttribsBeforeMethodToken(Tokens $tokens, $index)
      * @param Tokens $tokens Tokens collection
      * @param int    $index  token index
      *
-     * @return array array of grabbed attributes
+     * @return array<string, Token|null> array of grabbed attributes
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: this syntax is not supported by our current fixers

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: it's also not widely accepted as a standard

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: the same way that PSR-5 says don't use `@var`, but we all do anyway

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: > this syntax is not supported by our current fixers

all is green


> it's also not widely accepted as a standard

accepted by PSR5 proposal


> the same way that PSR-5 says don't use `@var`, but we all do anyway

what?
https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#824-var

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: It seems it really has been a while since I read the spec. Seems that change was reverted on 8th August 2014, lol. https://github.com/phpDocumentor/fig-standards/commit/61c695c43d1b2c713acbfb3302f284c518421bc5

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: I think we should remove the var_to_type fixer in 2.x then since `@type` isn't a thing anymore.

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: feel free to send a PR

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: i'll add an issue for it

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: i'd like to do that after the other stuff has gone through and the branches have been merged

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: aren't they merged?

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: no?

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: which one?

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1487 and/or https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1512.

The 2nd one replaces the first one, but is breaking. I'm now suggesting take the non-breaking parts from the first, then apply them only to 1.11, then take the rest, and target the master.

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: oh, say you means PRs, not origin branches :D

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: yeh :P

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: well, both i guess :D

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: nope ;P

File: Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php [hunk 2]
```diff
@@ -78,26 +78,26 @@ public function testFixProvider()
             // multi line array
             array(
                 '<?php $x = [  "a" =>$a,
-                    "b"=>  
+                    "b"=>
```
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php]: Are you sure having the trailing white space is not by design? (i.e. it tesst it is supported by the fixer being tested?)

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php]: I considered it. The fixer isn't touching it and doesn't care 'bout it.

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php]: I see, but isn't the point of testing making sure that stays that way?

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php]: This cases tests completely different thing, I can't see any benefits of that spaces.

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php]: :+1:  very well

File: Symfony/CS/Fixer/PSR2/VisibilityFixer.php (hunk 1 above)
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: still, i think that needs reverting - we shouldn't be using syntax our own code doesn't support

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: Then we should start supporting it

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: Extracted to separate PR: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1531

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/PSR2/VisibilityFixer.php]: > Then we should start supporting it

Easier said than done, but should be possible on the master branch when I'm finally done with the types stuff.
